### ISSUE TITLE
VizHub Android App

### ISSUE THREAD

**Issue by curran** [datavis-tech/vizhub-issue-tracker#602]

**Comment by stushurik** [comment]
@curran I think the best approach would be app based on web view, that way we would be able to have android and ios apps based on web version and use the same code. The company I work at using this approach and it works great

**Comment by curran** [comment]
Yes, totally! Probably wrapping the existing VizHub app using https://cordova.apache.org/ would be the most straightforward approach. I've never used Cordova myself.
